### DISCUSSION TITLE
希望翻译字幕的文件添加一些可选项

### DISCUSSION THREAD

**Discussion by ASmallAnimal** [Ayanaminn/N46Whisper#37] [Ideas]
目前使用GPT生成的字幕文件是双语字幕，但是据我所测试的情况来看，日语直接翻译成中文的效果并不是很好，但是翻译成英文的效果还不错，所以我希望增加单生成目标语言字幕文件的选项，这样我后续可以再处理比如英文字幕再转换成中文字幕，这样效果更好一些

**Comment by ASmallAnimal** [comment]
另外是否每次都要重复加载模型呢？不然计算单元有点吃不消啊，开通了pro一个月限额100个计算单元

**Reply by ASmallAnimal** [reply]
我稍微改了一代码块，把加载模型独立出来，防止重复加载避免占用内存和GPU资源

**Reply by Ayanaminn** [reply]
可以提个PR？
